ui/modules/organismes/Televersement.tsx

```diff
@@ -566,3 +573,24 @@ function InfoBetaPanel() {
     </Ribbons>
   );
 }
+
+function Header({ header }: { header: string }) {
+  if (headerTooltips[header]) {
+    return (
+      <>
+        {header}
+        <Tooltip background="bluefrance" color="white" label={<Box padding="2w">{headerTooltips[header]}</Box>}>
+          <Box
+            as="i"
+            className="ri-information-line"
+            fontSize="epsilon"
+            color="grey.500"
+            marginLeft="1v"
+            verticalAlign="middle"
+          />
+        </Tooltip>
+      </>
+    );
+  }
+  return <>header</>;
```

Comment:
```suggestion
  return <>{header}</>;
```
Je pense, sinon
![image](https://github.com/mission-apprentissage/flux-retour-cfas/assets/8938024/fc40368c-905c-4c35-9775-b0ab7c9ae654)

Reply:
Question bonus : c'est pas plus simple sinon de mettre directement un {headerTooltips[header] && (<Tooltip ...)} plutôt que de créer un autre composant ?

Reply:
Ah oui en effet, c'est pas mal non plus, je ne sais pas ce qui est le mieux (et merci pour le fix)